doc/design/garbage-collection-assets.md

```diff
@@ -0,0 +1,40 @@
+# Asset Garbage Collection
+
+## Background
+
+Now that the [design for asset blob and upload garbage collection](./garbage-collection-uploads-asset-blobs-2.md) is deployed to staging, we are ready to implement Asset garbage collection. The garbage collection process in this design document only applies to “regular” assets; garbage collection of zarrs is not covered and is out of scope for this design doc.
+
+## What is an “Orphaned Asset”
+
+It’s important to precisely define what it means for an asset to be eligible for garbage collection (an “orphaned asset”).
+An orphaned asset has the following properties:
+- The asset is not currently associated with any Versions
+- The asset is older than 30 days
+
+## Integration with existing garbage collection routines
+
+Garbage collection routines for Asset Blobs and Uploads are already implemented and deployed (though not run yet). Currently, the number of Asset Blobs and Uploads that need to be garbage collected is not substantial. However, the act of running the Asset garbage collection routine will result in a significant amount of Asset Blobs being orphaned, which can then in turn be collected by the Asset Blob garbage collection routine.
```

Comment:
interesting... so we do not even know on staging if blobs/uploads GC works?  

why not to deploy them in stages and not to enable already so we could decouple two changes and have more times for observing that first stage GC in action for longer?

Reply:
This is pretty much how we plan to roll this out. First, run the blob GC on sandbox, and make sure it works as expected. Next, after asset GC is implemented and merged (as described in this design doc), we can run asset GC on sandbox, watch that it exposes the expected asset _blobs_, then run the blob GC again to make sure this works "end to end".

We can add a section to the design doc that describes this approach.